src/store.ts

```diff
@@ -158,14 +192,40 @@ export default class Store {
     }
 
     if (resetDB) {
```

Comment:
do we need to add removal of the template files on reset, too?

Reply:
Good catch.